qemu/tests/chardev_tls_encryption.py

```diff
@@ -33,6 +34,9 @@ def run(test, params, env):
     """
     clean_cmd = params["clean_cmd"]
     try:
+        pkgs = params["depends_pkgs"].split()
```

Comment:
Suggest `params.get("depends_pkgs")` to avoid KeyError.

Reply:
Updated it.